.github/dependabot.yml

```diff
@@ -1,7 +1,25 @@
 version: 2
 updates:
   - package-ecosystem: github-actions
-    directory: "/"
+    directory: /
     schedule:
       interval: daily
```

Comment:
should we schedule all "weekly", just to have it consistent?

Reply:
Sounds good!